recipes-mmx/mmx-ep/mmx-ep.bb

```diff
@@ -0,0 +1,120 @@
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# NOTE
+#
+# This is part of a management middleware software package called MMX that was developed by Inango Systems Ltd.
+#
+# This version of MMX provides web and command-line management interfaces.
+#
+# Please contact us at Inango at support@inango-systems.com if you would like to hear more about
+# - other management packages, such as SNMP, TR-069 or Netconf
+# - how we can extend the data model to support all parts of your system
+# - professional sub-contract and customization services
+#
+################################################################################
+
+DESCRIPTION = "MMX Entry Point"
+HOMEPAGE = "https://github.com/InangoSystems/mmx-ep"
+LICENSE = "BSD-2-Clause-Patent"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b43fbcd44ec443ceba0ba4ee19519fd"
+SECTION = "mng"
+DEPENDS += "libmmx-frontapi libmmx-backapi"
+RDEPENDS_${PN} += "lua5.1 libmmx-frontapi libmmx-backapi libing-gen-utils"
+
+PV = "2.0.1"
+SRC_URI = "\
+    git://github.com/InangoSystems/mmx-ep.git;protocol=https;name=src;destsuffix=mmx-ep-${PV} \
+    git://github.com/InangoSystems/feed-mmx.git;protocol=https;branch=master;name=feed;subpath=net/mmx-ep/files;destsuffix=feed-mmx-ep-${PV} \
+"
+SRCREV_src = "9db7775ab0d1b97af61e7770f4d4785e9544bfaa"
+SRCREV_feed = "5172c6f7719cb5988f2722442238f400245fb564"
+
+
+S = "${WORKDIR}/mmx-ep-${PV}"
+S_FEED = "${WORKDIR}/feed-mmx-ep-${PV}"
+
+do_compile() {
+    oe_runmake
+}
+
+FILES_${PN} += "/usr/lib/lua/mmx"
+
+do_install() {
+    oe_runmake install DESTDIR=${D}
+
+    install -d ${D}${sbindir}
+    install -m 0755 ${S_FEED}/mmxep_env.sh ${D}${sbindir}
+
+    install -d ${D}/etc/init.d
+    install -m 0755 ${S_FEED}/mmx_extinit.sh ${D}/etc/init.d/mmx_extinit
+    install -m 0755 ${S_FEED}/mmxep_init.sh ${D}/etc/init.d/mmxep_init
+
+    # use to work without ing-tmp-overlay package
+    install -m 0755 ${S_FEED}/mmx_save_cfgfiles.sh ${D}${sbindir}
+    install -m 0755 ${S_FEED}/mmx_restore_cfgfiles.sh ${D}${sbindir}
+
+    install -d ${D}/etc/mmx/saved-config
+    install -d ${D}/etc/mmx/saved-db
+    install -d ${D}/etc/mmx/cand-db
+
+    install -d ${D}/usr/lib/lua/mmx
+    install -m 0755 ${S_FEED}/lua/mmx_db_reformat_utils.lua ${D}/usr/lib/lua/mmx/mmx_db_reformat_utils.lua
+    install -m 0755 ${S_FEED}/lua/mmx_db_reformat_perform.lua ${D}/usr/lib/lua/mmx/mmx_db_reformat_perform.lua
+    install -m 0755 ${S_FEED}/lua/mmx_db_get_version.lua ${D}/usr/lib/lua/mmx/mmx_db_reformat_get_version.lua
```

Comment:
typo or intentional change from `mmx_db_get_version.lua` to `mmx_db_reformat_get_version.lua`

Reply:
This line has been ported  from [original mmx-ep feed](https://github.com/InangoSystems/feed-mmx/blob/f4c6fb722639ea625d3e2a5d094dc925bb3c4bf8/net/mmx-ep/Makefile#L243) without paths modifications
I've rechecked that SVN version of `inango-openwrt/feeds/net/mmx-ep/Makefile` contains the same paths.

Reply:
ACK